website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -127,6 +127,8 @@ To leave an organization, click the ellipses (**...**) next to the organization
 
 Click **Password** in the sidebar to change your password.
```

Comment:
I love this suggestion, but we try not to add callouts unless absolutely necessary, so I moved your suggestion to the paragraph above and made some edits to adhere to our style guide: 
```suggestion
Click **Password** in the sidebar to change your password. You cannot manage your HCP Terraform password directly if you linked your account to GitHub or HashiCorp Cloud Platform (HCP). Instead, manage your password through the linked service.
```